source/docs/networking/networking-introduction/roborio-network-troubleshooting.rst

```diff
@@ -1,61 +1,101 @@
 roboRIO Network Troubleshooting
 ===============================
 
-The roboRIO and the 2015 FRC tools use dynamic IP addresses (DHCP) for network connectivity. This article describes steps for troubleshooting networking connectivity between your PC and your roboRIO
+The roboRIO and the 2015 and later FRC tools use dynamic IP addresses (DHCP) for network connectivity. This article describes steps for troubleshooting networking connectivity between your PC and your roboRIO
```

Comment:
```suggestion
The roboRIO and FRC tools use dynamic IP addresses (DHCP) for network connectivity. This article describes steps for troubleshooting networking connectivity between your PC and your roboRIO
```

I don't think we need to state 2015, as we use a yearly versioning model for the docs now.